src/components/form/switch/_switch.scss

```
@@ -1,33 +1,20 @@
 .euiSwitch {
+  $euiColorSwitchIsOff: lightOrDarkTheme(transparentize($euiColorMediumShade, .8), transparentize($euiColorMediumShade, .3));
```

We are housing these switch variables the `_variables.scss` within the `form/` component's folder. I'd move this one there as well and rename it to align with the other names.

```suggestion
  $euiSwitchOffColor: lightOrDarkTheme(transparentize($euiColorMediumShade, .8), transparentize($euiColorMediumShade, .3));
```